src/simplewallet/simplewallet.cpp

```diff
@@ -7670,11 +7685,13 @@ bool simple_wallet::sweep_below(const std::vector<std::string> &args_)
   if (args_.size() < 1)
   {
     fail_msg_writer() << tr("missing threshold amount");
+    PRINT_USAGE(USAGE_SWEEP_BELOW);
     return true;
   }
   if (!cryptonote::parse_amount(below, args_[0]))
   {
     fail_msg_writer() << tr("invalid amount threshold");
+    PRINT_USAGE(USAGE_SWEEP_BELOW);
```

Comment:
I would remove this as it doesn't fit the "print usage when given no args" commit title and also isn't consistent when entering an invalid amount in other places.